### DISCUSSION TITLE
对于多个add_packages，有什么便捷的方法修改cflags?

### DISCUSSION THREAD

**Discussion by Zmmfly** [xmake-io/xmake#2616] [General]
本地库可以在上级xmake.lua中，用add_deps添加phony属性的target实现批量修改cflags
对于package有什么好方法吗？

**Comment by waruqi** [comment]
add_requires("xxx", {configs = {cflags = "xxx"}})

**Reply by Zmmfly** [reply]
收到，非常感谢！

有没有一种可能:
add_packages("xx", {deps="mgr"})

这样，cflags交给mgr统一管理，即使添加的package非常多，
也能分行写，而不用每分一行就得复制一份

**Reply by waruqi** [reply]
add_requireconfs("*", {configs = {cflags = "xxx"}})

**Reply by Zmmfly** [reply]
感谢大佬，这下可以愉快地用xmake撸stm32了
